rcp/org.testeditor.rcp4.views.tests/src/org/testeditor/rcp4/views/tests/tree/AmlModelLabelProviderTest.xtend

```diff
@@ -0,0 +1,91 @@
+package org.testeditor.rcp4.views.tests.tree
+
+import javax.inject.Inject
+import org.eclipse.xtext.xbase.lib.Procedures.Procedure1
+import org.junit.Test
+import org.testeditor.aml.AmlFactory
+import org.testeditor.aml.TemplateText
+import org.testeditor.aml.TemplateVariable
+import org.testeditor.rcp4.views.AmlModelLabelProvider
+
+class AmlModelLabelProviderTest extends AbstractParserTest {
+
+	@Inject
+	AmlModelLabelProvider amlModelLabelProvider
+
+// TODO: get this injected!
+//	@Inject
+//	AmlFactory factory
+	AmlFactory factory = AmlFactory.eINSTANCE
+
+	@Test
+	def void labelForModelIsPackage() {
```

Comment:
Just my personal preference:

I think it's more readable to always use the with operator when initializing objects in the // when section like this:

    factory.createAmlModel => [
        package = "test"
    ]

And if the `// when` and `// then` sections are trivial and test a side-effect free method I usually combine then to an `// expect` (got this notation from the [Spock](http://spockframework.github.io/spock/docs/1.0/spock_primer.html) framework)